subsys/bluetooth/services/ots/ots_oacp.c

```diff
@@ -161,6 +163,53 @@ static enum bt_gatt_ots_oacp_res_code oacp_delete_proc_validate(
 }
 #endif
 
+#if defined(CONFIG_BT_OTS_OACP_CHECKSUM_SUPPORT)
+static enum bt_gatt_ots_oacp_res_code oacp_checksum_proc_validate(
+	struct bt_conn *conn,
+	struct bt_ots *ots,
+	struct bt_gatt_ots_oacp_proc *proc)
+{
+	struct bt_gatt_ots_oacp_cs_calc_params *params = &proc->cs_calc_params;
```

Comment:
```suggestion
	const struct bt_gatt_ots_oacp_proc *proc)
{
	const struct bt_gatt_ots_oacp_cs_calc_params *params = &proc->cs_calc_params;
```

Reply:
Const qualifier can't be assigned since member checksum_resp of params will be changed in this function.

Reply:
Why do we store the result in the parameters? That seems like a misuse of the word "parameter".

Reply:
You're right. I misused that parameter to store result.
Add function parameter to retrieve checksum sum.